layouts/partials/nav.html

```diff
@@ -64,10 +64,55 @@
             {{ end }}
 
             <li class="ms-2">
-                <button class="btn donate-btn btn-outline-primary text-uppercase px-4 rounded-0"
-                        type="button" style="font-size: 0.9rem;">
+                <button class="btn donate-btn btn-outline-primary text-uppercase px-4 rounded-0" type="button" data-bs-toggle="modal" data-bs-target="#donateModal" style="font-size: 0.9rem;">
                     Donate
-                </button>
+                  </button>
+                  
+                  <!-- Donate Modal -->
```

Comment:
![image](https://github.com/user-attachments/assets/257497bd-28c5-427d-8911-317d077e7e99)

The content in modal may get too long in one day, i would suggest implementing scrollable inner content.

You might find vertically centered modal useful as well.